### ISSUE TITLE
cannot install any packges using conda

### ISSUE THREAD

**Issue by Khalilsqu** [conda/conda#8926]
## Current Behavior

```
C:\Users\kalho>conda install notebook
Collecting package metadata (current_repodata.json): done
Solving environment: failed

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "C:\Miniconda3\lib\site-packages\conda\exceptions.py", line 1043, in __call__
        return func(*args, **kwargs)
      File "C:\Miniconda3\lib\site-packages\conda\cli\main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "C:\Miniconda3\lib\site-packages\conda\cli\conda_argparse.py", line 82, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "C:\Miniconda3\lib\site-packages\conda\cli\main_install.py", line 20, in execute
        install(args, parser, 'install')
      File "C:\Miniconda3\lib\site-packages\conda\cli\install.py", line 280, in install
        force_reinstall=context.force_reinstall or context.force,
      File "C:\Miniconda3\lib\site-packages\conda\core\solve.py", line 112, in solve_for_transaction
        force_remove, force_reinstall)
      File "C:\Miniconda3\lib\site-packages\conda\core\solve.py", line 150, in solve_for_diff
        force_remove)
      File "C:\Miniconda3\lib\site-packages\conda\core\solve.py", line 257, in solve_final_state
        ssc = self._check_solution(ssc)
      File "C:\Miniconda3\lib\site-packages\conda\core\solve.py", line 896, in _check_solution
        assert not any(d.name == spec.name for d in ssc.solution_precs)
    AssertionError

`$ C:\Miniconda3\Scripts\conda-script.py install notebook`

  environment variables:
                 CIO_TEST=<not set>
                CONDA_EXE=C:\Miniconda3\condabin\..\Scripts\conda.exe
               CONDA_EXES="C:\Miniconda3\condabin\..\Scripts\conda.exe"
               CONDA_ROOT=C:\Miniconda3
                 HOMEPATH=\Users\kalho
                     PATH=C:\Miniconda3;C:\Miniconda3\Library\mingw-w64\bin;C:\Miniconda3\Librar
                          y\usr\bin;C:\Miniconda3\Library\bin;C:\Miniconda3\Scripts;C:\Miniconda
                          3\bin;C:\Miniconda3;C:\Miniconda3\Library\mingw-w64\bin;C:\Miniconda3\
                          Library\usr\bin;C:\Miniconda3\Library\bin;C:\Miniconda3\Scripts;C:\Pyt
                          hon36\Scripts\;C:\Python36\;C:\Program Files\Microsoft MPI\Bin\;C:\Pyt
                          hon37\;C:\Python37\Scripts\;C:\Windows\system32;C:\Windows;C:\Windows\
                          System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\S
                          ystem32\OpenSSH\;C:\Program Files (x86)\NVIDIA
                          Corporation\PhysX\Common;C:\Program Files\Microsoft
                          MPI\Bin;C:\Users\kalho\AppData\Local\Programs\Common\Microsoft\Visual
                          C++ for Python\9.0;C:\Users\kalho\AppData\Local\Programs\Common\Micros
                          oft\Visual C++ for Python\9.0\VC\bin;C:\Program
                          Files\MATLAB\R2018b\runtime\win64;C:\Program
                          Files\MATLAB\R2018b\bin;C:\Program Files\MATLAB\MATLAB
                          Runtime\v94\runtime\win64;C:\Program Files\NVIDIA Corporation\NVIDIA N
                          vDLISR;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WIND
                          OWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\P
                          rogram Files (x86)\Calibre2\;C:\Users\kalho\AppData\Local\Microsoft\Wi
                          ndowsApps;C:\Users\kalho\AppData\Local\Programs\Microsoft VS Code\bin;
                          ;C:\Users\kalho\AppData\Local\Microsoft\WindowsApps;C:\texlive\2018\bi
                          n\win32;C:\Miniconda3\lib\site-packages\pywin32_system32
             PSMODULEPATH=C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\Windows
                          PowerShell\v1.0\Modules
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : None
       user config file : C:\Users\kalho\.condarc
 populated config files : C:\Users\kalho\.condarc
          conda version : 4.7.5
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : __cuda=10.1
       base environment : C:\Miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Miniconda3\pkgs
                          C:\Users\kalho\.conda\pkgs
                          C:\Users\kalho\AppData\Local\conda\conda\pkgs
       envs directories : C:\Miniconda3\envs
                          C:\Users\kalho\.conda\envs
                          C:\Users\kalho\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.7.5 requests/2.22.0 CPython/3.7.3 Windows/10 Windows/10.0.18362
          administrator : False
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.


```
### Steps to Reproduce

```
conda install <package>
```

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

     active environment : None
       user config file : C:\Users\kalho\.condarc
 populated config files : C:\Users\kalho\.condarc
          conda version : 4.7.5
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : __cuda=10.1
       base environment : C:\Miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Miniconda3\pkgs
                          C:\Users\kalho\.conda\pkgs
                          C:\Users\kalho\AppData\Local\conda\conda\pkgs
       envs directories : C:\Miniconda3\envs
                          C:\Users\kalho\.conda\envs
                          C:\Users\kalho\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.7.5 requests/2.22.0 CPython/3.7.3 Windows/10 Windows/10.0.18362
          administrator : False
             netrc file : None
           offline mode : False
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

==> C:\Users\kalho\.condarc <==
pinned_packages:
  - defaults::conda
ssl_verify: True
channels:
  - conda-forge
  - defaults
report_errors: False
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

# packages in environment at C:\Miniconda3:
#
# Name                    Version                   Build  Channel
asn1crypto                0.24.0                py37_1003    conda-forge
basemap                   1.2.0            py37h79c95a4_4    conda-forge
blas                      1.0                         mkl    defaults
bzip2                     1.0.6             hfa6e2cd_1002    conda-forge
ca-certificates           2019.6.16            hecc5488_0    conda-forge
certifi                   2019.6.16                py37_0    conda-forge
cffi                      1.12.3           py37hb32ad35_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.0                      pypi_0    pypi
conda                     4.7.5                    py37_0    conda-forge
conda-package-handling    1.3.10                   py37_0    conda-forge
cryptography              2.7              py37hb32ad35_0    conda-forge
cycler                    0.10.0                     py_1    conda-forge
decorator                 4.4.0                      py_0    conda-forge
fastcache                 1.1.0            py37hfa6e2cd_0    conda-forge
freetype                  2.10.0               h5db478b_0    conda-forge
future                    0.17.1                py37_1000    conda-forge
geos                      3.7.2                habb2df7_1    conda-forge
helpdev                   0.6.10                   pypi_0    pypi
icc_rt                    2019.0.0             h0cc432a_1    defaults
icu                       58.1                     vc14_0    conda-forge
idna                      2.8                   py37_1000    conda-forge
importlib-metadata        0.18                     pypi_0    pypi
inflection                0.3.1                    pypi_0    pypi
intel-openmp              2019.4                      245    defaults
jpeg                      9c                hfa6e2cd_1001    conda-forge
kiwisolver                1.1.0            py37he980bc4_0    conda-forge
libarchive                3.3.3             h4890af2_1006    conda-forge
libblas                   3.8.0                     8_mkl    conda-forge
libcblas                  3.8.0                     8_mkl    conda-forge
libiconv                  1.15              hfa6e2cd_1005    conda-forge
liblapack                 3.8.0                     8_mkl    conda-forge
libpng                    1.6.37               h7602738_0    conda-forge
libxml2                   2.9.9                h9ce36c8_1    conda-forge
libxslt                   1.1.32            heafd4d3_1003    conda-forge
lxml                      4.3.4            py37heafd4d3_0    conda-forge
lz4-c                     1.8.3             he025d50_1001    conda-forge
lzo                       2.10              hfa6e2cd_1000    conda-forge
matplotlib                3.1.1                    py37_0    conda-forge
matplotlib-base           3.1.1            py37h2852a4a_0    conda-forge
menuinst                  1.4.16                   py37_0    conda-forge
mkl                       2019.4                      245    defaults
mpmath                    1.1.0                      py_0    conda-forge
numpy                     1.16.4           py37hc71023c_0    conda-forge
obspy                     1.1.1            py37h452e1ab_0    conda-forge
openssl                   1.1.1c               hfa6e2cd_0    conda-forge
pandas                    0.24.2           py37he350917_0    conda-forge
peakutils                 1.3.2                      py_0    conda-forge
pip                       19.1.1                   py37_0    conda-forge
proj4                     4.9.3                hcf24537_7    defaults
psutil                    5.6.3                    pypi_0    pypi
pycosat                   0.6.3           py37hfa6e2cd_1001    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyopenssl                 19.0.0                   py37_0    conda-forge
pyparsing                 2.4.0                      py_0    conda-forge
pyproj                    1.9.5.1         py37haf3c679_1006    conda-forge
pyqt                      5.9.2            py37h6538335_0    conda-forge
pyqt5                     5.11.3                   pypi_0    pypi
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqt5-tools               5.11.3.1.4               pypi_0    pypi
pyshp                     2.1.0                      py_0    conda-forge
pysocks                   1.7.0                    py37_0    conda-forge
python                    3.7.3                h8c8aaf0_0    defaults
python-dateutil           2.8.0                      py_0    conda-forge
python-dotenv             0.10.3                   pypi_0    pypi
python-libarchive-c       2.8                   py37_1004    conda-forge
pytz                      2019.1                     py_0    conda-forge
pywin32                   224             py37hfa6e2cd_1000    conda-forge
qdarkstyle                2.7                      pypi_0    pypi
qt                        5.9.7                hc6833c9_1    conda-forge
requests                  2.22.0                   py37_0    conda-forge
ruamel_yaml               0.15.71         py37hfa6e2cd_1000    conda-forge
scipy                     1.2.1            py37h29ff71c_0    defaults
setuptools                41.0.1                   py37_0    conda-forge
sip                       4.19.8          py37h6538335_1000    conda-forge
six                       1.12.0                py37_1000    conda-forge
sqlalchemy                1.3.5            py37hfa6e2cd_0    conda-forge
sqlite                    3.29.0               hfa6e2cd_0    conda-forge
sympy                     1.4                      py37_0    conda-forge
tinycss                   0.4                      pypi_0    pypi
tornado                   6.0.3            py37hfa6e2cd_0    conda-forge
tqdm                      4.32.2                     py_0    conda-forge
urllib3                   1.24.3                   py37_0    conda-forge
vc                        14.1                 h0510ff6_4    defaults
vs2015_runtime            14.15.26706          h3a45250_4    defaults
wheel                     0.33.4                   py37_0    conda-forge
win_inet_pton             1.1.0                    py37_0    conda-forge
wincertstore              0.2                   py37_1002    conda-forge
xz                        5.2.4             h2fa13f4_1001    conda-forge
yaml                      0.1.7             hfa6e2cd_1001    conda-forge
zipp                      0.5.2                    pypi_0    pypi
zlib                      1.2.11            h2fa13f4_1004    conda-forge
zstd                      1.4.0                hd8a0e53_0    conda-forge
</p></details>

**Comment by Khalilsqu** [comment]
`conda install -c anaconda notebook` gives the same assertion error

**Comment by msarahan** [comment]
The error is coming from your pinned_packages configuration.  I'm not sure why - that may be a bug.  Remove that configuration and try again.  The bug should be fixed in 4.7.6 (on conda-canary) or 4.7.7 (soon to be released)

**Comment by Khalilsqu** [comment]
what command should I run to remove the configuration,  >> `conda config --set restore_free_channel false`?

**Comment by msarahan** [comment]
Edit the file.  That's all ``conda config`` does.  The file is listed in the output you pasted above: ``C:\Users\kalho\.condarc``

**Comment by msarahan** [comment]
I added that configuration locally.  I didn't see an error.  It must be that setting in combination with some other state of your system.  We'll get the next version of conda out and see if this is still an issue.

**Comment by Khalilsqu** [comment]
![image](https://user-images.githubusercontent.com/22060413/61164075-10b40500-a50a-11e9-94d2-724e10138ef9.png)

This are the file contents. Not sure what to edit exactly.

Thanks anyway. I will be waiting the the new version. really appreciate your help. Good luck all.

**Comment by Khalilsqu** [comment]
Okay, it is working right now, I just re-installed miniconda and updated to conda 4.7.7. I also deleted the files .conda and .condarc. thgen right now I can install packages as normal.

Thanks a lot.

**Comment by liuyigh** [comment]
I had same error on 4.7.5. I had plotly pinned to version 3. After removing that pinned line, I am able to update conda and everything again

**Comment by caifazhou** [comment]
I got this erorr on windows 10:

ProxyError: Conda cannot proceed due to an error in your proxy configuration.
Check for typos and other configuration errors in any '.netrc' file in your home directory,
any environment variables ending in '_PROXY', and any other system-wide proxy
configuration settings.

with anaconda version 4.10.0

Totally no idea why it happens because I have configured .condarc. The .netrc file should be the one for linux if I understood correctly.
